src/dstack/_internal/cli/services/configurators/run.py

```diff
@@ -165,65 +171,16 @@ def apply_configuration(
         abort_at_exit = False
         try:
             # We can attach to run multiple times if it goes from running to pending (retried).
-            while True:
-                with console.status(f"Launching [code]{run.name}[/]") as status:
-                    while run.status in (
-                        RunStatus.SUBMITTED,
-                        RunStatus.PENDING,
-                        RunStatus.PROVISIONING,
-                    ):
-                        job_statuses = "\n".join(
-                            f"  - {job.job_spec.job_name} [secondary]({job.job_submissions[-1].status.value})[/]"
-                            for job in run._run.jobs
-                        )
-                        status.update(
-                            f"Launching [code]{run.name}[/] [secondary]({run.status.value})[/]\n{job_statuses}"
-                        )
-                        time.sleep(5)
-                        run.refresh()
-                console.print(
-                    f"[code]{run.name}[/] provisioning completed [secondary]({run.status.value})[/]"
-                )
-
-                current_job_submission = run._run.latest_job_submission
-                if run.status in (RunStatus.RUNNING, RunStatus.DONE):
-                    if run._run.run_spec.configuration.type == RunConfigurationType.SERVICE.value:
-                        console.print(
-                            f"Service is published at [link={run.service_url}]{run.service_url}[/]\n"
-                        )
-                    bind_address: Optional[str] = getattr(
-                        configurator_args, _BIND_ADDRESS_ARG, None
-                    )
-                    try:
-                        if run.attach(bind_address=bind_address):
-                            for entry in run.logs():
-                                sys.stdout.buffer.write(entry)
-                                sys.stdout.buffer.flush()
-                        else:
-                            console.print("[error]Failed to attach, exiting...[/]")
-                            exit(1)
-                    finally:
-                        run.detach()
-
-                # After reading the logs, the run may not be marked as finished immediately.
-                # Give the run some time to transit into a finished state before exiting.
-                reattach = False
-                for _ in range(30):
-                    run.refresh()
-                    if _run_resubmitted(run, current_job_submission):
-                        # The run was resubmitted
-                        reattach = True
+            # Live table setup
+            with Live(console=console, refresh_per_second=LIVE_TABLE_REFRESH_RATE_PER_SEC) as live:
+                while True:
+                    live.update(self.get_runs_table([run], verbose=True))
```

Comment:
Cannot access attribute "get_runs_table" for class "BaseRunConfigurator*"

Ensure get_runs_table is properly imported and used.